src/stringProcessing/directPrecedenceException.js

```diff
@@ -0,0 +1,30 @@
+var getWordIndices = require( "../researches/passivevoice/getIndicesWithRegex.js" );
```

Comment:
I'm not sure whether this is the right location for this file (and related files). Why aren't they in the passive voice folder/language specific folder?

Reply:
You mean `directPrecedenceException.js`? I put it in this folder because it's used for both English and French.